pkg/compactor/compactor.go

```diff
@@ -181,5 +181,8 @@ func (cp *Compactor) Compact(ctx context.Context, opts *brtypes.CompactOptions)
 		cancel()
 	}
 
+	// Add a sleep command so that prometheus can collect necessary metrics related to the uploading of snapshots. see https://github.com/gardener/etcd-druid/issues/648
+	time.Sleep(60 * time.Second)
```

Comment:
Since  a context is passed to this method a better way to achieve this would be:
```go
func SleepWithContext(ctx context.Context, sleepFor time.Duration) error {
	for {
		select {
		case <-ctx.Done():
			return ctx.Err()
		case <-time.After(sleepFor):
			return nil
		}
	}
}
```

Reply:
Done